### PR TITLE
kinto command for init and start

### PR DESCRIPTION
**Pull request by Natim**
Superseed #232

### CODE REVIEW

**Comment by Natim** [comment]
@leplatrem final r?

**Comment by Natim** [comment]
We should have the config_file parameter for the other command as well.

File: kinto/__main__.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        if args['which'] == 'init':
+                if(args['config_file'] is None):
+                    env = bootstrap('config/kinto.ini')
+                else:
+                    config_file = format(args['config_file'])
+                    env = bootstrap(config_file)
+        elif args['which'] == 'migrate':
+                env = bootstrap('config/kinto.ini')
+                cliquet.init_schema(env)
```
Comment (leplatrem) [inline kinto/__main__.py]: I would suggest to use a local variable `config_file` everywhere.

From what I read here, `migrate` is always executed with `config/kinto.ini`

**Comment by leplatrem** [comment]
r+

**Comment by leplatrem** [comment]
(*don't forget to update the contributors ;)*)

**Comment by Natim** [comment]
> (don't forget to update the contributors ;))

It can be done at release time isn't it?

**Pull request merged**